shared/keystore/key.go

```diff
@@ -129,10 +129,10 @@ func (k *Key) UnmarshalJSON(j []byte) (err error) {
 	}
 
 	if err := k.PublicKey.Deserialize(pubkey); err != nil {
-		return err
+		return fmt.Errorf("Unable to deserialize public key: %v", err)
```

Comment:
Do not capitalize errors

Reply:
i think that was in an old commit, its not capitalized now